20220610.md

```diff
@@ -45,3 +45,124 @@ This last fortnight, over the 17 pull requests or activities reported in the ope
+
+The obvious competitor to SailBabel is Sebastian Wolf's (**WerkWolf**) Wunderfitz. While Wunderfitz has more features, with its scan-translations, toolbar and in-built dictionary list, there's an elegant simplicity to SailBabel that also makes it feel more responsive and lighter to use.
+
+When it comes to either app, the experience is ultimately tied to the quality of the database in use. I found translating between Finnish and English using the [dict.cc](http://www1.dict.cc) database to be a bit hit-and-miss. Often the stem or *vartalo* word exists is in the database, but because neither app supports fuzzy search, getting at it wasn't always obvious.
+
+These are still both really nice apps to have for Sailfish OS, and if you want an offline dictionary you should definitely given SailBabel a try. It's available from the Jolla Store.
+
+## Picross 2
+
+We've covered Matti Viljanen's (**direc85**) Picross 2 on several occassions in the past, and it still holds up as one of my favourite puzzle games for Sailfish OS. Officially described as nonogram, but in my experience often referred to as Hanjie puzzles, you must fill in the grid of squares with either blocks or crosses, using only the numbers around the edge of the grid as clues. Picross 2 offers 103 hand-crafted puzzles with a gentle difficulty curve that starts with some easy tuturial and ends up with the aptly titled "Insane" levels. If you have time to spare, you could do worse than unspare it playing Picross 2. The latest version adds support for [My Backup](https://openrepos.net/content/slava/my-backup) and makes some adjustments to the user interface and translations. Version 2.5.0 is available from the Jolla Store and [OpenRepos](https://openrepos.net/content/direc85/picross-2 ).
+
+## Frome
+
+Rustem Abzalov (**arustg**) has been prolific in updating apps this fortnight. From is a simple app that makes it easy to calculate percentages, useful if you're calculating interest or tips. The interface is unusual but intuitive, with a the buttons displayed as a row in pairs down the left or &mdash; with a configuration option &mdash; right hand side. A history of calculations then fills up on the edge without the numbers.
```

Comment:
/with a the buttons/with the buttons/ without the 'a' maybe.

Reply:
Thanks, nice spot. Should now be fixed.